kobo/apps/accounts/tests/test_social_account.py

```diff
@@ -29,3 +34,61 @@ def test_delete(self):
         res = self.client.delete(url)
         self.assertEqual(res.status_code, 204)
         self.assertFalse(self.user.socialaccount_set.exists())
+
+
+class ProviderTestCase(APITestCase):
+    def setUp(self):
+        # modify settings to add test social account provider
+        with contextlib.ExitStack() as exit_stack:
```

Comment:
Can override_settings be used instead?

Reply:
Thanks, didn't notice this can be used as a class decorator.